src/components/v5/common/ActionSidebar/ActionSidebar.tsx

```diff
@@ -18,19 +18,103 @@ import PopularActions from './partials/PopularActions';
 import useToggle from '~hooks/useToggle';
 import { ACTION_TYPE_FIELD_NAME } from './consts';
 import Modal from '~v5/shared/Modal';
+import { ActionForm } from '~shared/Fields';
+import { ActionFormBaseProps } from './types';
+import { formatText } from '~utils/intl';
 
 const displayName = 'v5.common.ActionSidebar';
 
-const ActionSidebar: FC<PropsWithChildren> = ({ children }) => {
+interface Props extends ActionFormBaseProps {
+  toggleIsSidebarFullscreen: () => void;
+  isSidebarFullscreen: boolean;
+}
+
+const ActionSidebarFormContent: FC<PropsWithChildren<Props>> = ({
+  children,
+  toggleIsSidebarFullscreen,
+  isSidebarFullscreen,
+  getFormOptions,
+}) => {
   const {
-    form,
-    selectedAction,
-    hasErrors,
     formComponent: FormComponent,
-    getFormOptions,
-    onSubmit,
-  } = useActionForm();
-  const intl = useIntl();
+    hasErrors,
+    selectedAction,
+  } = useSidebarActionForm();
+  const isMobile = useMobile();
+  const {
+    actionSidebarToggle: [, { toggle: toggleActionSidebarOff }],
+  } = useActionSidebarContext();
+  const userHasPermissions = useUserHasPermissions();
+  const form = useFormContext();
+  const notificationBanner = useNotificationBanner(hasErrors, selectedAction);
+
+  return (
+    <>
+      <div className="py-4 px-6 flex w-full items-center justify-between border-b border-gray-200">
+        {isMobile ? (
+          <button
+            type="button"
+            className="py-2.5 flex items-center justify-center text-gray-400"
+            onClick={toggleActionSidebarOff}
+            aria-label={formatText({ id: 'ariaLabel.closeModal' })}
+          >
+            <Icon name="close" appearance={{ size: 'tiny' }} />
+          </button>
+        ) : (
+          <button
+            type="button"
+            className="py-2.5 flex items-center justify-center text-gray-400"
+            onClick={toggleIsSidebarFullscreen}
+            aria-label={formatText({ id: 'ariaLabel.fullWidth' })}
+          >
+            <Icon
+              name={
+                isSidebarFullscreen ? 'arrow-right-line' : 'arrows-out-simple'
+              }
+              appearance={{ size: 'tiny' }}
+            />
+          </button>
+        )}
+        {children}
+      </div>
+      <div className="px-6 py-8 flex-grow">
+        <input
+          type="text"
+          className={`
+            heading-3 placeholder:text-gray-500
+            hover:text-blue-400 hover:placeholder:text-blue-400 text-gray-900
```

Comment:
Please add hovers only on desktop - Ik it's my code 💀

Reply:
`md:` added